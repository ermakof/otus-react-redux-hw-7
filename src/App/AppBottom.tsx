import React, { FC, memo } from 'react';
import Panel from '@src/layout/Panel';

const AppBottom: FC = () => {
  return (
    <Panel>
      <p role="userCount">Результат</p>
    </Panel>
  );
};

export default memo(AppBottom);
