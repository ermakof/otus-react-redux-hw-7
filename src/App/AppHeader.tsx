import React, { FC, memo } from 'react';
import Panel from '@src/layout/Panel';

const AppHeader: FC = () => {
  return (
    <Panel>
      <p>Game of Life</p>
    </Panel>
  );
};

export default memo(AppHeader);
