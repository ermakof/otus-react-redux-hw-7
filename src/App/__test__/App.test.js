import React from 'react';

import { render } from '@testing-library/react';
import App from '@src/App';

test('render App', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
