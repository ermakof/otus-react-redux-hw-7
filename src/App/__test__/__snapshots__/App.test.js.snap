// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App 1`] = `
<DocumentFragment>
  <div
    class="css-5simcc"
  >
    <div
      class="css-1co368a"
    >
      <p>
        Game of Life
      </p>
    </div>
    <div
      class="css-xt4orx"
      role="dataList"
    >
      <section
        class="css-hsjawn"
        role="cellsGrid"
      >
        <div
          class="css-1vrw23c"
          role="cellContainer-0"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-x1m0eg"
          role="cellContainer-1"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-re5xc0"
          role="cellContainer-2"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-1vrw23c"
          role="cellContainer-3"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-x1m0eg"
          role="cellContainer-4"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-re5xc0"
          role="cellContainer-5"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-1vrw23c"
          role="cellContainer-6"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-x1m0eg"
          role="cellContainer-7"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          class="css-re5xc0"
          role="cellContainer-8"
        >
          <span
            class="css-b7hmd9"
            role="cellContent"
          />
        </div>
        <div
          style="clear: both;"
        />
      </section>
    </div>
    <div
      class="css-1co368a"
    >
      <p
        role="userCount"
      >
        Результат
      </p>
    </div>
  </div>
</DocumentFragment>
`;
